rdr_service/dao/consent_dao.py

```diff
@@ -32,6 +31,11 @@ def get_participants_with_consents_in_range(cls, session, start_date,
                 ParticipantSummary.email.notlike('%@example.com')
```

Comment:
For future consideration:  I've been thinking that we might be better off not doing additional screening to identify test participants that PTSC hasn't told us about, so we don't mask the fact that they haven't told us.  That's assuming we have an agreement that we will no longer have to check multiple conditions on our end (e.g., HPO pairing, too?) in addition to the `is_test_participant` flag in the `participant` table.   Would really like to rely on that one field if we can.